### PR TITLE
readOnlyRootFilesystem changed to true

### PR DESCRIPTION
**Pull request by NitishKumar06**
#34117

### CODE REVIEW

File: content/en/examples/policy/restricted-psp.yaml [hunk 1]
```diff
@@ -45,4 +45,5 @@ spec:
       # Forbid adding the root group.
       - min: 1
         max: 65535
-  readOnlyRootFilesystem: false
+  readOnlyRootFilesystem: true
+
```
Comment (Sea-n) [inline content/en/examples/policy/restricted-psp.yaml:49]: Are there any reason to add a empty line in the end of file?

**Comment by NitishKumar06** [comment]
No reason as such. Shall I remove the extra line?

**Comment by tengqm** [comment]
I'd like to see some evidences why this change is needed.

**Comment by reylejano** [comment]
@NitishKumar06 , please take a look at @tengqm 's [comment](I'd like to see some evidences why this change is needed.)

File: content/en/examples/policy/restricted-psp.yaml (hunk 1 above)
Comment (reylejano) [inline content/en/examples/policy/restricted-psp.yaml:49]:
```suggestion
  readOnlyRootFilesystem: true
```

**Comment by NitishKumar06** [comment]
Hey! I think this was not an issue to be worked upon. Shall I close this PR?

**Comment by zacharysarah** [comment]
@NitishKumar06 👋🏻 
> I think this was not an issue to be worked upon. Shall I close this PR?

As wrangler for the week, I can do it for you.
/close

**Comment by NitishKumar06** [comment]
Thank you ! @zacharysarah

**Pull request closed without merging**